threedsecure/client_test.go

```diff
@@ -32,22 +33,32 @@ func TestThreeDSecureNew(t *testing.T) {
 		ReturnURL: "https://test.com",
 	}
 
-	target, err := New(threeDSecureParams)
+	tdsrc, err := New(threeDSecureParams)
```

Comment:
Good point. I used `tdsrc` because it's the prefix for this resource's ID, but I'm not too sure what it stands for either!

I replaced with `threeDSecure` per your suggestion.